identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -54,13 +54,13 @@ <h2 class="form__heading">Newsletter options</h2>
             <div class="fieldset__fields email-subscription__options">
                 <ul class="u-unstyled">
                     <li>
-                        <a type="button" href=@buildIdentityUrl("account/edit") class="manage-account__button manage-account__button--mini " data-link-name="Change email address">Update your email address</a>
+                        <a role="link" href="@buildIdentityUrl("account/edit")" class="manage-account__button manage-account__button--mini" data-link-name="Change email address">Update your email address</a>
                     </li>
                     <li>
-                        <div type="link" role="link" class="js-email-subscription__formatFieldsetToggle manage-account__button manage-account__button--mini " data-link-name="Change format">Change format</div>
+                        <a href="#manage-account__modal--newsletterFormat" role="button" class="js-email-subscription__formatFieldsetToggle manage-account__button manage-account__button--mini" data-link-name="Change format">Change format</a>
                     </li>
                     <li>
-                        <label class="label">No longer want to receive any of these emails?</label>
+                        <p class="label-like">No longer want to receive any of these emails?</p>
                         <button type="button" class="manage-account__button--mini manage-account__button email-unsubscribe js-unsubscribe" data-link-name="Unsubscribe from all emails"><span class="email-unsubscribe-all__label js-unsubscribe--basic">Unsubscribe from all</span><span class="email-unsubscribe-all__label js-unsubscribe--confirm hide">Are you sure?</span></button>
```

Comment:
What about this button? Screenread would read is as

`Unsubscribe from allAre you sure?`

The wording could be enhanced (unsubscribe from what?) plus `Are you sure?` could be moved into `role="alert"` ...